Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -78,6 +78,10 @@ void BreakPoints::AddFromStrings(const TBreakPointsStr& bp_strings)
 
     iss >> std::hex >> bp.address;
     iss >> flags;
+    std::getline(iss, bp.message);
+    if (!bp.message.empty() && bp.message.front() == ' ')
```

Comment:
Dumb question...is there a reason we only erase the first space?  Should we use `StripSpaces` instead?